### PR TITLE
add page with travel info

### PR DESCRIPTION
**Pull request by marcoow**
localhost_1111_travel

### CODE REVIEW

File: content/travel.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
++++
+template = "generic-content.html"
+title = "Traveling to Vienna and the Venue"
++++
+
+<h3 class="mb-3 mt-7">To Vienna</h3>
+<div class="box text-centered">
+  <p>Vienna is well reachable via train from many cities in Europe. Also, Vienna is the central hub of ÖBB's extensive network of sleeper trains. If you're looking for an eco-friendly way to travel to EuroRust, check out the <a href="https://www.nightjet.com/en" target="_blank">nightjets</a>. <strong class="nightjet-warning">Make sure to book early though – nightjets tend to be fully booked months in advance!</strong></p>
+  <br>
+  <p>If there's no good train connection from the place you're traveling from or you're looking for a more price-conscious alternative, there's buses as well, e.g. via <a href="https://www.omio.com/buses" target="_blank">Omio</a>.</p>
```
Comment (emmasofiah2o) [inline content/travel.md:10]:
```suggestion
  <p>If there's no good train connection from the place you're traveling from or you're looking for a more price-conscious alternative, there are buses as well, e.g., via <a href="https://www.omio.com/buses" target="_blank">Omio</a>.</p>
```

File: content/travel.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <br>
+  <p>Vienna also has an international airport, <a href="https://www.viennaairport.com/en" target="_blank">VIE</a>, if you'd rather fly in.</p>
+</div>
+
+<h3 class="mb-3 mt-7">To the Venue</h3>
+<div class="box text-centered">
+  <p>The venue is located just outside of the city center. The best options to get there from the city center are buses 72A, 76A, 76B, or train line S80. Bus tickets can be bought via <a href="https://www.wienerlinien.at/web/wl-en" target="_blank">Wiener Linien</a>. Tickets for the train can be bought via <a href="https://shop.oebbtickets.at/en/ticket" target="_blank">ÖBB</a> or at the stations.</p>
```
Comment (emmasofiah2o) [inline content/travel.md:17]:
```suggestion
  <p>The venue is located just outside of the city center. The best options to get there from the city center are buses 72A, 76A, 76B, or the train line S80. Bus tickets can be bought via <a href="https://www.wienerlinien.at/web/wl-en" target="_blank">Wiener Linien</a>. Tickets for the train can be bought via <a href="https://shop.oebbtickets.at/en/ticket" target="_blank">ÖBB</a> or at the stations.</p>
```

File: content/travel.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <p>The venue is located at <a href="https://maps.app.goo.gl/pM1kFhCqWGrSyvaF9" target="_blank">2. Haidequerstraße 1–3, 1110 Wien</a>. Once you're there, enter the area via the main gate and follow the EuroRust signs to the conference venue:</p>
+  <br>
+  <img src="/images/entrance.jpg" class="venue-entry">
+</div>
+
+<h3 class="mb-3 mt-7">Assistance, Questions, Visas</h3>
+<div class="box text-centered">
+  <p>For any questions or in case you need assistance planning your travel or getting to the venue, please reach out at <a href="mailto:mail@eurorust.eu">mail@eurorust.eu</a></p>
+  <br>
+  <p>If you require a Visa to go to Austria and need support, e.g. by receiving an invitation letter, please reach out as well.</p>
```
Comment (emmasofiah2o) [inline content/travel.md:28]:
```suggestion
  <p>If you require a Visa to go to Austria and need support, e.g., by receiving an invitation letter, please reach out as well.</p>
```

**Review by emmasofiah2o — APPROVED** [review]
Approving already, but I left a few minimal corrections

File: content/travel.md (hunk 3 above)
Comment (marcoow) [reply content/travel.md:28]: I think this would be wrong

**Pull request merged**